code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -527,3 +527,45 @@
 /obj/effect/mob_spawn/human/oldsci/Destroy()
 	new/obj/structure/showcase/machinery/oldpod/used(drop_location())
 	return ..()
+
+
+#define PIRATE_NAMES_FILE "pirates.json"
+
+/obj/effect/mob_spawn/human/pirate
+	name = "space pirate sleeper"
+	desc = "A cryo sleeper smelling faintly of rum."
+	random = TRUE
+	icon = 'icons/obj/cryogenic2.dmi'
+	icon_state = "sleeper"
+	mob_name = "a space pirate"
+	mob_species = /datum/species/human
+	outfit = /datum/outfit/pirate
+	roundstart = FALSE
+	death = FALSE
+	anchored = TRUE
+	density = FALSE
+	flavour_text = "<font size=3><b>Y</b></font><b>ou are an space pirate. The station refused to pay for your protection, protect the ship,siphon the credits from the station and raid it for even more loot.</b>"
+	assignedrole = "Space Pirate"
+	var/rank = "Mate"
+
+/obj/effect/mob_spawn/human/pirate/special(mob/living/new_spawn)
+	new_spawn.real_name = generate_pirate_name()
+
+/obj/effect/mob_spawn/human/pirate/proc/generate_pirate_name()
+	var/beggings = strings(PIRATE_NAMES_FILE, "beginnings")
+	var/endings = strings(PIRATE_NAMES_FILE, "endings")
+	return "[rank] [pick(beggings)][pick(endings)]"
+
+/obj/effect/mob_spawn/human/pirate/Destroy()
+	new/obj/structure/showcase/machinery/oldpod/used(drop_location())
+	return ..()
+
+/obj/effect/mob_spawn/human/pirate/Initialize(mapload)
+	. = ..()
+	notify_ghosts("Space pirates are waking up!", source = src, action=NOTIFY_ATTACK, flashwindow = FALSE, ignore_mapload = FALSE)
+
+/obj/effect/mob_spawn/human/pirate/captain
```

Comment:
give the captain the special pirate hat. the one pirates always wear, the tricorne-y thing.

Reply:
Right now they all get that because it's part of the spaceproof suit, but yeah probably better if only the others get spaceproof bandanas